app/models/osbb.rb

```diff
@@ -0,0 +1,14 @@
+class Osbb < ApplicationRecord
+  before_save :downcase_email
+  
+  validates :name, presence: true, length: { minimum: 3, maximum: 255 }
+  validates :phone, :postcode,  numericality: { only_integer: true }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
move contstant to the top(ruby style quide

Reply:
Resolve